frontend/src/features/admin/users/ManageUsers.test.tsx

```diff
@@ -103,12 +106,7 @@ describe('Manage Users Component', () => {
         },
       });
     });
-    expect(getAllByRole('option')[0]).toHaveTextContent('agencyVal');
-  });
-
-  it('Does not display disabled agencies', () => {
-    const { queryByText } = testRender(getStore());
-    expect(queryByText('disabledAgency')).toBeNull();
```

Comment:
For whatever reason the test can't actually see the dropdown options with ParentSelect, so this test wasn't actually testing anything

Reply:
something to try in situations like this is dumping the entire dom with prettydom (something like console.log(prettyDOM(container, 99999))

alternatively sometimes you need to wait for your condition to be true:
```
wait(() => {
//wait will run and expectations in a loop until it times out or the expectation stops failing.
  expect(true).toBeTruthy();
});
```